tests/integration/test_repos_repo.py

```diff
@@ -718,6 +718,17 @@ def test_ignore(self):
             subscription = repository.ignore()
             assert subscription.ignored is True
 
+    def test_invitations(self):
+        """Test that a user can iterate over the invitations to a repo."""
+        self.token_login()
+        cassette_name = self.cassette_name('invitations')
+        with self.recorder.use_cassette(cassette_name):
+            repository = self.gh.repository('jacquerie', 'flask-shell-bpython')
```

Comment:
This is because I needed a repository for which I have admin rights, otherwise it returns a 403.